main.nf

```diff
@@ -23,17 +23,19 @@ def helpMessage() {
     nextflow run nf-core/epitopeprediction -profile <docker/singularity/conda/institute> --input "*.vcf.gz"
 
     Mandatory arguments:
-      --input [file]                        Path to input data (must be surrounded with quotes)
-      --alleles [file]                      Path to the file containing the MHC alleles
       -profile [str]                        Configuration profile to use. Can use multiple (comma separated)
                                             Available: conda, docker, singularity, test, awsbatch, <institute> and more
 
+    Input:
+      --input [file]                        Path to input data (must be surrounded with quotes)
+      --alleles [file]                      Path to the file containing the MHC alleles
+    
     Alternative inputs:
       --peptides [file]                     Path to TSV file containing peptide sequences (minimum required: id and sequence column)
       --proteins [file]                     Path to FASTA file containing protein sequences
 
     Pipeline options:
-      --show_supported_models [str]         Writes out supported models. Does not run actual prediction pipeline. Do not specify this in combination with other input files. Default: false.
+      --show_supported_models [str]         Writes out supported models. Does not run actual prediction pipeline (mutually exclusive with input files). Default: false.
```

Comment:
Sorry, I don't get why it has to be mutually exclusive. In the end it doesn't matter if someone still specifies `input` right?

Reply:
Hm.. I just wanted to avoid confusion. I didn't see why someone would want to run the actual pipeline in combination with `--show_supported_models`. On the other hand, I thought it might be misleading if some parameters would be just ignored, what would the user want then, running the prediction or showing the supported models?  
But I can change that if you think, for example, the input should just be ignored (maybe in combination with an adjusted log info, stating that no prediction was run)?

Reply:
In my opinion that would suffice :)